<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UR</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QLabel" name="labelNetwork">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Listening:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNetworkPort">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>55056</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRobot1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Robot 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRobot1Status">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OFFLINE</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupRobot1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Robot 1</string>
   </property>
   <widget class="QLabel" name="labelBase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base:</string>
    </property>
   </widget>
   <widget class="QSlider" name="targetBase">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="actualBase">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualBase">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTargetBase">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QSlider" name="targetShoulder">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelShoulder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shoulder:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="actualShoulder">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualShoulder">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTargetShoulder">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="actualElbow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="targetElbow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualElbow">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelElbow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elbow:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTargetElbow">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="actualWrist1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualWrist1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QSlider" name="targetWrist1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelWrist1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrist1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTargetWrist1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualWrist2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="actualWrist2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelWrist2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrist2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTargetWrist2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QSlider" name="targetWrist2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelWrist3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wrist3:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="actualWrist3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="targetWrist3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualWrist3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTargetWrist3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00°</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
